pyOCD/target/target_lpc11u24.py

```diff
@@ -16,19 +16,17 @@
 """
 
 from cortex_m import CortexM
+from .memory_map import (FlashRegion, RamRegion, MemoryMap)
 
 class LPC11U24(CortexM):
 
-    memoryMapXML =  """<?xml version="1.0"?>
-<!DOCTYPE memory-map PUBLIC "+//IDN gnu.org//DTD GDB Memory Map V1.0//EN" "http://sourceware.org/gdb/gdb-memory-map.dtd">
-<memory-map>
-    <memory type="flash" start="0x0" length="0x8000"> <property name="blocksize">0x400</property></memory>
-    <memory type="ram" start="0x10000000" length="0x1000"> </memory>
-</memory-map>
-"""
-    
+    memoryMap = MemoryMap(
+        FlashRegion(    start=0,           length=0x8000,       blocksize=0x400, isBootMemory=True),
```

Comment:
The block size should be 0x1000.  That is the actual page_size used by flash_lpc11u24.py.